src/renderer/components/Editor/Editor.tsx

```diff
@@ -155,15 +158,20 @@ export default class Editor extends React.Component<Props> {
       const cursor = editor.getCursor();
       const token = editor.getTokenAt(cursor);
       const tokenString = token.string;
+      if (tokenString.length < MIN_COMPLETION_CHARS) {
+        return;
+      }
       CodeMirror.showHint(editor, undefined, {
-        hint: (): CodeMirror.Hints => {
+        hint: (cm: CodeMirror.Editor): CodeMirror.Hints => {
+          const tableHints = this.props.tables.filter(t => t.length > tokenString.length && t.startsWith(tokenString));
+          // Suppress 'hint does not exist' because codemirror does not serve types of addons.
+          // @ts-ignore
```

Comment:
Use `@ts-expect-error` instead. If you use `@ts-expect-error`, we can detect unnecessary annotations when CodeMirror provides the correct types in the future.